### PR TITLE
Update subgraphs for m2 contract changes

### PR DESCRIPTION
**Pull request by mooselumph**
## Checks

- [ ] I've made sure the lint is passing in this PR.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, in that case, please comment that they are not relevant.
- Testing Strategy
   - [ ] Unit tests
   - [ ] Integration tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: subgraphs/eigenda-operator-state/networks.json [hunk 1]
```diff
@@ -1,104 +1,88 @@
...
-    },
-    "BLSPubkeyRegistry_QuorumApkUpdates": {
+    "BLSApkRegistry_QuorumApkUpdates": {
       "address": "0x0000000000000000000000000000000000000000",
       "startBlock": 0
     }
   },
   "anvil": {
-    "BLSRegistryCoordinatorWithIndices": {
+    "RegistryCoordinator": {
```
Comment (jianoaix) [inline subgraphs/eigenda-operator-state/networks.json:25]: What about the addresses of these new contracts?

**Review by jianoaix — COMMENTED** [review]
should we remove https://github.com/Layr-Labs/eigenda/blob/master/subgraphs/eigenda-operator-state/src/bls-pubkey-compendium.ts ?

**Comment by jianoaix** [comment]
> should we remove https://github.com/Layr-Labs/eigenda/blob/master/subgraphs/eigenda-operator-state/src/bls-pubkey-compendium.ts ?

Also:
- https://github.com/Layr-Labs/eigenda/blob/master/subgraphs/eigenda-operator-state/src/bls-registry-coordinator-with-indices.ts
- https://github.com/Layr-Labs/eigenda/blob/master/subgraphs/eigenda-operator-state/src/bls-pubkey-registry.ts

File: subgraphs/eigenda-operator-state/networks.json (hunk 1 above)
Comment (mooselumph) [reply subgraphs/eigenda-operator-state/networks.json:25]: These are dummy addresses that get replaced by the deployer. Making that explicit by zeroing them out. 

It looks like the `preprod-goerli` and `goerli` ones may correspond to actual contracts on goerli, but we can't update those until the new deployment.

**Review by jianoaix — APPROVED** [review]

**Pull request merged**